src/Vts/MonteCarlo/VirtualBoundaries/LateralBoundingVirtualBoundary.cs

```diff
@@ -1,39 +1,30 @@
 ﻿using System;
-using System.Runtime.CompilerServices;
 using Vts.MonteCarlo.PhotonData;
 
 namespace Vts.MonteCarlo.VirtualBoundaries
 {
     /// <summary>
-    /// The <see cref="VirtualBoundaries"/> namespace contains the Monte Carlo virtual boundary classes 
+    /// Implements IVirtualBoundary.  Used to capture all photons absorbed by bounding 
+    /// tissue region (whatever is dependency injected)
     /// </summary>
-
-    [CompilerGenerated]
-    internal class NamespaceDoc
```

Comment:
We need to leave this in place, it is for the XML documentation

Reply:
I wasn't sure if I should move this to the first class in the folder.  I think it doesn't matter but due to class renaming, this class isn't the first in the folder anymore.

Reply:
I think it just needs to be in a file that has that namespace so it should be fine leaving it here.

Reply:
Ok, will put back in this class.